docs/installation/advanced/integrating-from-source-vite.md

```diff
@@ -144,6 +141,55 @@ ClassicEditor.defaultConfig = {
 };
 ```
 
+If you scaffolded your project using a Vite template with TypeScript, add the `ckeditor.ts` file in the `src` folder. Then, modify the file by adding the following lines of code.
```

Comment:
Same as in the webpack guide. Please add TypeScript and JavaScript headings to relevant code snippets.